app/views/invitation_mailer/invite.html.erb

```diff
@@ -0,0 +1,9 @@
+<p>Hi<%= @name && " #{@name}" %>,</p>
+<p>Thanks for joining the Ruby AU Slack Channel. We're super excited for you to join! We'd like to take this opportunity to invite you to become a member of the Ruby Australia website.<p>
+
+<p>If you're interested - <strong>it doesn't cost anything!</strong> - you can sign up below:</p>
+<p class="button"><%= link_to "Join", new_user_registration_url %></p>
+
+<h2>What does being a member mean?</h2>
+<p>As a member, you get to attend and vote at our general meetings, and be nominated (by yourself or others) for a position in the association's committee.</p>
+<p>We will also send you very occasional emails related to the association (perhaps a handful each year).</p>
```

Comment:
I wonder if we can use the same template for both types of emails here, to ensure we're not doubling up on copy too much? And maybe have a param to determine the context of why someone's being invited.